### PR TITLE
Added dry-run option to all commands that changes ssh config.

### PR DESCRIPTION
**Pull request by emre**

### CODE REVIEW

File: storm/__init__.py [hunk 1]
```diff
@@ -27,18 +27,23 @@ def __init__(self, ssh_config_file=None):
...
         if self.is_host_in(name):
             raise ValueError(ERRORS["already_in"].format(name))
 
         options = self.get_options(host, user, port, id_file, custom_options)
 
         self.ssh_config.add_host(name, options)
+
+        if dry_run:
+            return self.ssh_config.dump()
+
```
Comment (ddegirmenci) [inline storm/__init__.py]: You should have an `else` here.

Comment (f) [inline storm/__init__.py]: No, he doesn't. After two lines there's `return True`.

This is a valid use:
```python
if expression:
  return False
return True
```

This is same thing with:

```python
if expression:
  return False
else:
  return True
```

Comment (ddegirmenci) [inline storm/__init__.py]: Mmm yes, I missed the return before `self.ssh_config.dump()`. I thought that made the dry run kinda pointless. Makes sense now :)

Comment (emre) [inline storm/__init__.py]: 👍

**Pull request closed without merging**